app/js/authentication_controller.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+/* global SumoDB, User, Utils */
+
+(function(exports) {
+  var USER_CREDENTIALS_KEY = 'user_credentials';
+
+  function authenticate(evt) {
+    evt.preventDefault();
+
+    var form = document.getElementById('login');
+    var username = form.elements['username'].value;
+    var password = form.elements['password'].value;
+
+    var promise = SumoDB.authenticate_user(username, password);
```

Comment:
We should do User.authenticate_user and let the User model deal with calling SumoDB, storing the credentials, etc. It's what a model does :)